features/step_definitions/publisher_steps.rb

```diff
@@ -0,0 +1,48 @@
+When "I go to add an artefact" do
+  click_on "Add artefact"
+  expect(page).to have_content("New artefact")
+end
+
+When "I create a draft place" do
+  fill_in "Title", with: "Smokey test draft place"
+  fill_in "Slug", with: "smokey-test-draft-place"
+  select "Place", from: "Format"
+  click_button "Save and go to item"
+end
+
+When "I preview the draft place" do
+  # Wait 5 seconds before previewing, to give time for Router to update it's routes
+  sleep(5)
```

Comment:
I wonder if instead we could use `wait_until` (see: features/support/delay.rb) which might make this more robust (and potentially quicker!).

Reply:
I considered this at the time but actually I think you're right. Interesting that [`wait_until` was actually removed in Capybara v2.0](https://www.varvet.com/blog/why-wait_until-was-removed-from-capybara) though!